code/modules/cargo/packs/security.dm

```diff
@@ -357,3 +357,26 @@
 	cost = CARGO_CRATE_VALUE * 7
 	contains = list(/obj/item/storage/belt/holster/thermal = 2)
 	crate_name = "thermal pistol crate"
+	
+/datum/supply_pack/security/armory/wt550
+	name = "Recalled Weapon Pack"
+	desc = "Contains a set of old nanotranset brand autorifles recalled due to choking hazard."
+	cost = CARGO_CRATE_VALUE * 7
+	hidden = TRUE
+	contains = list(/obj/item/gun/ballistic/automatic/wt550,/obj/item/ammo_box/magazine/wt550m9,
+					/obj/item/ammo_box/magazine/wt550m9,
+					/obj/item/gun/ballistic/automatic/wt550)
+	crate_name = "crate"
```

Comment:
why

Reply:
you marked this as resolved but did not answer why

Reply:
I don't know what they were asking in the why part but this has been changed to fit the style of other organization. In short one of fikous suggestions broke it so I had to figure out which one did

Reply:
They are asking why you are doing `crate_name = "crate"`

Reply:
I think it was a relic part of me trying to get the crate to spawn correctly

Reply:
that should fix it